samples/WOCCatalog/WOCCatalog/URLSessionViewController.mm

```diff
@@ -107,19 +113,28 @@ - (void)loadView {
     [self.view addSubview:_blockSwitchContainer];
 
     UIButton* cancelLastButton = [UIButton buttonWithType:UIButtonTypeRoundedRect];
-    cancelLastButton.frame = (CGRect){ BUTTON_ROW_ITEM_X(nbuttons % BUTTON_NUM_IN_ROW), BUTTON_ROW_ITEM_Y(nbuttons++ / BUTTON_NUM_IN_ROW), BUTTON_ROW_ITEM_WIDTH, BUTTON_ROW_HEIGHT };
+    cancelLastButton.frame = (CGRect){ BUTTON_ROW_ITEM_X(nbuttons % BUTTON_NUM_IN_ROW),
+                                       BUTTON_ROW_ITEM_Y(nbuttons++ / BUTTON_NUM_IN_ROW),
+                                       BUTTON_ROW_ITEM_WIDTH,
+                                       BUTTON_ROW_HEIGHT };
```

Comment:
The `nbuttons++` here is execptionally important: it makes sure that every button is in a new slot.